README.md

```diff
@@ -320,19 +320,20 @@ With Git-Flow, branches are categorised into
 * One or several **feature** branches.
 * One or several **hotfix** branches.
 
-The code on the **master** branch (often called `master` or `release`) is
+The code on the **master** branch (often called `master`) is
```

Comment:
```suggestion
The code on the **main** branch (often called `main` or `release`) is
```

Reply:
@airenzp @nanjiangshu @norling what about this changes?

Reply:
I approve at least.